packages/ui-extensions/src/surfaces/admin/extension.ts

```diff
@@ -4,6 +4,22 @@ import type {ExtensionTargets} from './extension-targets';
 
 export * from '../../extension';
 
+export interface ShopifyGlobal {
```

Comment:
I think we can revert this change since we won't be exposing the data extension global